plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -497,6 +497,12 @@ CacheStats getStatisticsCacheStats()
         cache.invalidateAll(cacheKeys);
     }
 
+    @Override
+    public void fireDataChangedEvent(JdbcTableHandle handle)
```

Comment:
or `onDataChange` 

("fire" seems more appropriate to event dispatcher to me)

Reply:
"fire" and "on" are good prefixes for event bus, where one side triggers and other side responds. Here things are bundled together so neither of these two fits.